### PR TITLE
new style toolboxes lol

### PR DESCRIPTION
**Pull request by dannno**
:cl: Dannno
tweak: hahaha I switched your toolboxes you MORONS
/:cl:

_Mmm. Wake up Dr. It is time to get to work._

I shake the good doctor awake and rise from the table I had been sleeping on, my eyes turning immediately to the belt sander sitting in the corner of the room. Excitement fills my breast and a smile creeps across my face. There ain't nothing in this world I love more than fresh belt smell.

_What're we gonna do today, Adam?_

The good Dr. Chine sits on the edge of the table, rubbing his eyes sleepily.

_Well today I think our first step is going to be putting a new belt on the belt sander, Dr. Chine._

I squat down in front of one of my many work benches and pull out a cardboard box, dusting it off and opening it. Inside lies a plethora of fresh belts, the sight of which fills my heart with glee.

_There ain't NOTHING in this world that I love more than some fresh belt smell._

My hand hovers over the belt pile, savoring it, as my eyes take it all in. I pick out a nice one from the corner and replace the box. Giggling madly, I stand and move back to the belt sander. I carefully wrap the fresh belt around the mechanism. Gently I caress the machine once over to confirm that everything is as it should be... and then I start it up. It whirs to life, a delectable sound which I have grown to love.

_Come, Dr._

I beckon my good friend the venerable Dr. Chine to my side. After all, what good are the pleasures and flavors of life if you have no comrades to share them with? He sighs and walks over to the belt sander.

_Would you just get a WHIFF of that belt smell, Dr. Chine?_

I grab him by the back of his head firmly, taking him by surprise as I force his head down in front of the sander. He yelps in fear and tries to fight me, but I am simply too strong.

_Adam, please! This is too close!_

_Smell it, Dr. Take a good sniff of it._

_My hair is caught! OUCHIE!_

He cries out in pain and I let him free of my grasp. He tugs himself loose and backs off to tend to his wounds as I chuckle heartily. I wipe the tears from my eyes and bend down to get a good whiff of that belt smell, and my beard gets caught in the sander.

_Yoowwwwcchh!!!_

### CODE REVIEW

**Comment by coiax** [comment]
while you're here, make clown and mime toolboxes as well

**Comment by dannno** [comment]
Damnable conflicts.

**Comment by WJohn** [comment]
Neat. I like em.

**Comment by PKPenguin321** [comment]
They look shitty and small

**Comment by dannno** [comment]
so do you but you don't hear me crying about it

**Comment by PKPenguin321** [comment]
While this is true that's still a real issue with these sprites

**Comment by chanoc1** [comment]
thes look great

**Comment by PKPenguin321** [comment]
*small*

**Comment by dannno** [comment]
How is that a negative

**Comment by PKPenguin321** [comment]
They're tiny! Smaller than a toolbelt!

**Comment by TrustyGun** [comment]
Size does not matter, it's what you do with it

**Comment by Yoshmaster** [comment]
I kinda wanna lick those sprites
they look so clean and nice

**Comment by Militaires** [comment]
remind me of goon toolboxes

did you rip that off or are these actually new sprites

did u actually fucking sprtie nainything danno.>>/?u90
\?

**Comment by chanoc1** [comment]
what the fuck are these commit descriptions

**Comment by dannno** [comment]
they're edited goon toolboxes
emitting "goon" from the title was an attempt to spare the "GOON GO HOME" posts lol!

**Comment by Militaires** [comment]
@chanoc if u think those are bad check out my riding fix pr 

also goon is cool so yeah

**Comment by Xhuis** [comment]
Good sprites but the old ones are so classic. Eeeeeh.

**Comment by Hatfish** [comment]
tbh it'd look better if it were just our current toolboxes with new high-contrast colors like the oxygen tanks.

**Comment by chanoc1** [comment]
muh lega- bad meme

**Comment by Anonmare** [comment]
The red toolbox is just so classic, I can't let it go

**Comment by chanoc1** [comment]
reeeeeeeeeeeeeeeeeeeeeeeeee

**Comment by Fox-McCloud** [comment]
Those look great 👍

**Comment by GunHog** [comment]
They look nice!

**Comment by Fox-McCloud** [comment]
These are Goon's sprites, aren't they?

**Comment by MimicFaux** [comment]
New sprites are always welcome, but the toolbox is _iconic_ to SS13. My very first few steps in the game was picking one up, and then bludgeoning myself in the face accidentally trying to open it. Bleeding, I tried to heal myself with a medkit, and promptly clubbed myself in the face with that too.

I'm all for new sprites, but toolboxes are kinda sacred. Hatfish has the right of it, if they're to be updated, template the original sprites and spruce them up with high contrast colours.

**Comment by SweaterKittens** [comment]
I like them, nice job Danno. You also fixed the weird thing with the original goon ones where the bottom looked super tiny for some reason.

**Comment by dannno** [comment]
"iconic" is just an excuse in my opinion
we culture a community notoriously against change and I think this is just a symptom of it

if this is rejected, feel free to "spruce up" the current toolboxes yourself. I don't want to do that.

**Comment by PKPenguin321** [comment]
Why do you want small toolboxes

**Comment by dannno** [comment]
why do you want big shit that clutters everything
I understand that the workspace you have for these sprites is 32x32 but that doesn't mean everything has to take up all that fucking room, it makes such a mess
these toolboxes aren't even that much smaller than the existing ones, only like 2 pixels off the top and sides or something.

**Comment by chanoc1** [comment]
these are also shaded much better and their perspective is a lot more consistent with the rest of the sprites

**Comment by Armhulen** [comment]
:((((((((((((((((( i don't want the old ones to GOOOOOO

**Comment by Steelpoint** [comment]
The tool boxes are nice, but what is up with the OP?

**Comment by RemieRichards** [comment]
@dannno solution: add a blue (classic) old style toolbox, covered in rust, it can spawn in space ruins and stuff.

everyone's happy then.

**Comment by dannno** [comment]
here you go :3

**Comment by Ausops** [comment]
this is a theoretical question

would it be better to style all item on floor sprites so that they look like, say, desktop icons? as in they are very recognizable as the item they are but they aren't a part of the world like a front facing 3/4 perspective might make sprites seem like

I ask because the current one is an icon sorta sprite while the new one looks like a 'part of the world' due to it's natural looking perspective

**Comment by Cheridan** [comment]
I prefer the latter, it makes everything look much nicer when its all the same perspective. Granted we have to make some concessions, like how brain icons are larger than people's actual heads, etc.

**Comment by dannno** [comment]
@Ausops I think it's better to make things look like part of the world, because they really are.

**Comment by RemieRichards** [comment]
Unless it's a UI element, of course.

**Comment by PKPenguin321** [comment]
@dannno I think His Grace has an alternate animated sprite of it shaking around, did you convert that one too?

**Comment by dannno** [comment]
did it all, buddy boy 
no worries :))

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 1]
```diff
@@ -48,6 +52,10 @@
 	new /obj/item/device/analyzer(src)
 	new /obj/item/weapon/wirecutters(src)
 
+/obj/item/weapon/storage/toolbox/mechanical
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/toolbox.dm]: you forgot `/old`

Comment (dannno) [inline code/game/objects/items/weapons/storage/toolbox.dm]: hah fuck

Comment (dannno) [inline code/game/objects/items/weapons/storage/toolbox.dm]: ahahah that's not good at all

Comment (Xhuis) [inline code/game/objects/items/weapons/storage/toolbox.dm]: it's not gamebreaking, just make a patch ez

**Pull request merged**